delivery_carrier_label_postlogistics/postlogistics/web_service.py

```diff
@@ -174,7 +174,8 @@ def _prepare_recipient(self, picking):
         }
 
         if partner.street2:
-            recipient['AddressSuffix'] = partner.street2
+            recipient['Street'] = partner.street2
+            recipient['AddressSuffix'] = partner.street
```

Comment:
I don't see handling of case when street is set but not street2
check code on https://github.com/OCA/delivery-carrier/pull/275/files  there is handling of these case